src/components/Planning/PlanningTable/PlanningRow/ProjectRow/ProjectCell.tsx

```diff
@@ -30,6 +39,104 @@ const addActiveClassToProjectRow = (projectId: string) => {
   document.getElementById(`project-row-${projectId}`)?.classList.add('active');
 };
 
+const getCellTypeUpdateRequestData = (cell: IProjectCell, phase: string): IProjectRequest => {
+  const { type, startYear } = cell;
```

Comment:
Grab the projectEstDates here, since your refering to project.estDates.estPlanningEnd etc. 

const { type, startYear, projectEstDates: { estPlanningEnd } } = cell;

...

req.estPlanningEnd = removeYear(estPlanningEnd);

Reply:
prev and next could also be grabbed here and other frequently used variables.